### PR TITLE
Add redshift profiler [sc-5880]

### PR DESCRIPTION
**Pull request by elic-eon**
### 🤔 Why?

To extracts column-level data profiles from Redshift & PostgreSQL.

### 🤓 What?

- Add Redshift & PostgreSQL profile connectors.

### 🧪 Tested?

Run connectors locally, manually verify MCEs.

### CODE REVIEW

File: metaphor/snowflake/profile/extractor.py [hunk 1]
```diff
@@ -109,7 +110,7 @@ def _fetch_tables(
                 # exclude both view and temporary table
                 continue
 
-            full_name = SnowflakeExtractor.table_fullname(database, schema, name)
+            full_name = dataset_fullname(database, schema, name)
```
Comment (mars-lan) [inline metaphor/snowflake/profile/extractor.py:113]: Interesting. How comes this wasn't caught by mypy or linters in https://github.com/MetaphorData/connectors/pull/165? @alyiwang

Comment (alyiwang) [reply metaphor/snowflake/profile/extractor.py:113]: I guess if a file is not touched, the linters will ignore it?

Comment (mars-lan) [reply metaphor/snowflake/profile/extractor.py:113]: Actually seems like this is one of the cases that mypy fails to detect an obvious mistake.

File: pyproject.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "metaphor-connectors"
-version = "0.10.19"
+version = "0.10.20"
```
Comment (mars-lan) [inline pyproject.toml:3]: Please update.

**Review by mars-lan — COMMENTED** [review]
Otherwise LGTM

**Review by mars-lan — APPROVED** [review]
Great work!

**Pull request merged**